### ISSUE TITLE
Keyboard interaction with React-MaterialUI: buttons onClick event called several times when pushed with space/enter key

### ISSUE THREAD

**Issue by danielweck** [edrlab/readium-desktop#32]
This is a known bug in `material-ui` 0.20.0 (latest):
https://www.npmjs.com/package/material-ui

Issue:
https://github.com/mui-org/material-ui/issues/9344

Pull Request (already merged in their `master` branch):
https://github.com/mui-org/material-ui/pull/9439

v0.20.0 was published on December 4th:
https://github.com/mui-org/material-ui/commits/master

We will close this issue when a new release of `material-ui` is available.

PS: I personally check my NPM dependencies regularly using the `ncu` CLI:
https://www.npmjs.com/package/npm-check-updates
(alternatively, the David web app is great too, e.g.: https://david-dm.org/edrlab/r2-streamer-js )

**Comment by danielweck** [comment]
I tested the OPDS dialog again, when using the ENTER / RETURN key to accept and close the dialog, the feed is added **6 times**! :)

**Comment by danielweck** [comment]
Question: this is now fixed for the OPDS dialog (by replacing MaterialUI's `FlatButton` with an "in-house" implementation that mimics the style, see commit https://github.com/edrlab/readium-desktop/pull/126/commits/9b017c36843b109b71211dd433e1dbd8c2d35e47 of in-progress Pull Request #126), but I wonder if the other (many) uses of `FlatButton` in the current codebase of ReadiumDesktop should be fixed too? Perhaps it would be easier if there was a custom `<FlatButton />` implementation in `readium-desktop` (keep the same name, but instead of `import FlatButton from "material-ui/FlatButton";` use `import FlatButton from "readium-desktop/ui/FlatButton";`)?
